drivers/usb/device/usb_dc_stm32.c

```diff
@@ -309,6 +317,20 @@ int usb_dc_attach(void)
 		return ret;
 	}
 
+	/* Enable VDDUSB if available */
+#ifdef PWR_CR2_PVME1
+	if (LL_APB1_GRP1_IsEnabledClock(LL_APB1_GRP1_PERIPH_PWR)) {
+		/* LL_PWR_EnableVddUSB(); */
```

Comment:
I would add the following in `arch/arm/soc/st_stm32/stm32l4/soc.h` to include `stm32l4xx_ll_pwr.h`:
```
#ifdef CONFIG_USB
/* Needed to remove USB OTG peripheral electrical isolation */
#include <stm32l4xx_ll_pwr.h>
#endif
```